daemon/daemon_unix.go

```diff
@@ -111,6 +111,10 @@ func getMemoryResources(config containertypes.Resources) *specs.LinuxMemory {
 		memory.Kernel = &config.KernelMemory
 	}
 
+	if config.KernelMemoryTCP > 0 {
```

Comment:
`!= 0` as in KernelMemory?

Reply:
@AkihiroSuda The PR has been updated. Please take a look.